api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedUserRouter.java

```diff
@@ -99,6 +118,22 @@ private void handleGetAllUsers(RoutingContext ctx) {
     end(ctx.response(), 200, JsonObject.mapFrom(users).toString());
   }
 
+  private void handleGetDisabledUsers(RoutingContext ctx) {
+    JWTData jwtData = ctx.get("jwt_data");
+    Optional<String> countryName =
+        RestFunctions.getOptionalQueryParam(ctx, "country", (str -> str));
+
+    UserListResponse users;
+
+    if (!countryName.isPresent()) {
+      users = processor.getDisabledUsers(jwtData, null);
```

Comment:
Do we have to pass null here? Is it possible to just write a version of this method with just the first argument?